lib/ui/widget/login_form.dart

```diff
@@ -100,12 +115,41 @@ class LoginFormState extends ConsumerState<LoginForm> {
               ),
             ),
             child: Text(AppLocalizations.of(context)!.login),
-            onPressed: () => {
-              // TODO: log in
-            },
+            onPressed: () => {_logIn()},
           ),
         ),
       ],
     );
   }
+
+  String? _emailValidator(String? value) {
+    if (value == null || !EmailValidator.validate(value)) {
+      return AppLocalizations.of(context)!.errorInvalidEmail;
+    } else {
+      return null;
+    }
+  }
+
+  String? _passwordValidator(String? value) {
+    if (value == null || value.length < _minPasswordLength) {
```

Comment:
Same here

Reply:
Resolved in https://github.com/nimblehq/ic-flutter-avishek/pull/36/commits/87733e49c74585d40d26767b5e685a67875506a2